projects/plugins/starter-plugin/jest.config.js

```diff
@@ -0,0 +1,8 @@
+const baseConfig = require( 'jetpack-js-tools/jest/config.base.js' );
+
+module.exports = {
+	...baseConfig,
+	roots: [ '<rootDir>/src' ],
+	moduleDirectories: [ 'node_modules', '<rootDir>/src' ],
```

Comment:
As I understand it, this only makes sense if you're doing like `import * from 'something'` instead of `import * from './something'` for local code. I'm not sure whether that's a pattern we actually want to encourage.

Reply:
once again, blind copy-paste :D. I'll remove this line completely, then.
```suggestion
	roots: [ '<rootDir>/src' ],
```